service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisCaseManagementOrder.java

```diff
@@ -1,47 +1,21 @@
 package uk.gov.hmcts.reform.fpl.model.docmosis;
 
-import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.experimental.SuperBuilder;
 import uk.gov.hmcts.reform.fpl.model.common.Schedule;
 
 import java.util.List;
 
 @Getter
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
 public class DocmosisCaseManagementOrder extends DocmosisOrder {
     private final List<DocmosisRepresentative> representatives;
     private final boolean scheduleProvided;
     private final Schedule schedule;
     private final List<DocmosisRecital> recitals;
     private final boolean recitalsProvided;
-
-    @Builder
-    public DocmosisCaseManagementOrder(DocmosisJudgeAndLegalAdvisor judgeAndLegalAdvisor,
-                                       String courtName,
-                                       String familyManCaseNumber,
-                                       String dateOfIssue,
-                                       String complianceDeadline,
-                                       List<DocmosisRespondent> respondents,
-                                       List<DocmosisChild> children,
-                                       boolean respondentsProvided,
-                                       String applicantName,
-                                       DocmosisHearingBooking hearingBooking,
-                                       List<DocmosisDirection> directions,
-                                       String draftbackground,
-                                       String courtseal,
-                                       String crest,
-                                       List<DocmosisRepresentative> representatives,
-                                       Schedule schedule,
-                                       boolean scheduleProvided,
-                                       List<DocmosisRecital> recitals,
-                                       boolean recitalsProvided,
-                                       int numberOfChildren) {
-        super(judgeAndLegalAdvisor, courtName, familyManCaseNumber, dateOfIssue, complianceDeadline, respondents,
-            children, respondentsProvided, applicantName, hearingBooking, directions, draftbackground, courtseal,
-            crest);
-        this.representatives = representatives;
-        this.schedule = schedule;
-        this.scheduleProvided = scheduleProvided;
-        this.recitals = recitals;
-        this.recitalsProvided = recitalsProvided;
-    }
+    private final DocmosisJudgeAndLegalAdvisor allocatedJudgeAndLegalAdvisor;
+    private final int numberOfChildren;
```

Comment:
I don't think that this field is needed, isn't used in the document

Reply:
thanks @lewisbirks

Reply:
I'm guessing the other new field is from your other story? Might be best not to have it in this one as it is strange that you are adding fields when just replacing the constructor.